scripts/Oluwaseun-Adeniran.py

```diff
@@ -0,0 +1 @@
+print(" Hello world , this is Adeniran Oluwaseun with HNGi7 ID HNG- 03483 and email adeniranoluwaseun0608@gmail.com  using python for stage 2 task")
```

Comment:
remove the space in between `HNG- 03483`

Reply:
and the space after World